### DISCUSSION TITLE
Apache isn't available in portable version?

### DISCUSSION THREAD

**Discussion by mh010** [leokhoa/laragon#568] [Q&A] [answered]
Using the portable version of Laragon, when I go to the Services & Ports tab in Preferences, the Apache line is greyed out. The checkbox to the left is checked, which I would think indicates that it's installed, but the Enabled box to the right is unchecked and after a little searching here and with Google, I still don't see how to change this so it's enabled. Would someone please enlighten me?

**Comment by Pen-y-Fan** [comment, marked as answer]
The portable version ships with Nginx. 

You can download Apache from [apache lounge](https://www.apachelounge.com/download/additional/), it needs to be the same Visual Studio C++ (VC) version as the version of PHP. e.g. PHP 7.4 is VC15, PHP 5.4 is VC9.

Extract the zip to laragon\bin\apache\

e.g. C:\laragon\bin\apache\httpd-2.4.54-win64-VS16

Then restart Laragon to auto-detect it.

**Reply by mh010** [reply]
Ok, thank you.
